src/Directory.Build.props

```diff
@@ -43,9 +43,13 @@
     <ManagedSourceCode>$(ProjectDir)Microsoft.Data.SqlClient\</ManagedSourceCode>
     <SqlServerSourceCode>$(ProjectDir)Microsoft.SqlServer.Server\</SqlServerSourceCode>
     <NetCoreSource>$(ManagedSourceCode)netcore\</NetCoreSource>
+	  <NetCoreResources>$(ManagedSourceCode)src\Resources\</NetCoreResources>
```

Comment:
Indent violation.

Reply:
Removed indentation. Not sure how it got indented.

Reply:
it's still there.

Reply:
It was lining up in the editor. But upon close inspection, they were indented using tabs. The tabs were replaced with spaces now.  Since the new lines were copied and pasted, it seemed the editor have replaced spaces with tabs automatically when pasted.